docs/pages/access-controls/access-requests/oss-role-requests.mdx

```diff
@@ -0,0 +1,159 @@
+including Resource Access Requests and an intuitive and searchable UI are
+available in Teleport Enterprise.
+
+## RBAC Setup
+
+Teleport's role-based access control (RBAC) allows you to configure what roles
+users can request access to. In this example, we will define two roles:
+
+1. `contractor`: users with this role can request elevated access to the `dba` role
+2. `dba`: this role grants access to databases
+
+There is no role for request approvers, because request approval rules can only
+be configured for Teleport Enterprise. In Open Source Teleport, approvals must
+be performed by running `tctl` on the Auth Server.
+
+**Contractor Role**
+
+Users with this role can request access to the `dba` role.
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: contractor
+spec:
+  allow:
+    request:
+      roles: ['dba']
+```
+
+**DBA Role**
+
+This role grants access to databases.
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: dba
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+  options:
+    # Only allows the contractor to use this role for 1 hour from time of request.
+    max_session_ttl: 1h
+```
+
+## Requesting Access
+
+In OSS Teleport, requests are made from the `tsh` CLI. To create an access
+request, use the `tsh request create` command.
+
+```code
+$ tsh request create \
+    --roles=dba \
+    --reviewers=bob \
+    --reason="performing DB migration tonight"
+```
+
+By default, this command will block until the request is approved. To submit the
+request without waiting for approval, add the `--nowait` flag.
+
+Alternatively, `tsh` can automatically create an access request during the login
+process. To activate this behavior, specify the `--request-roles` flag:
+
+```code
+$ tsh login --user=alice --request-roles=dba
+# Seeking request approval... (id: bc8ca931-fec9-4b15-9a6f-20c13c5641a9)
+```
+
+This will wait for the request to be approved, and then issue credentials with
+the `dba` role automatically when the request is approved.
+
+To log in and submit the request without waiting for approval, add the
+`--request-nowait` flag. In this scenario, you will receive your regular roles
+upon login, and can elevate your access after the request is approved.
+
+```code
+# log in with an approved access request
+$ tsh login --request-id=bc8ca931-fec9-4b15-9a6f-20c13c5641a9
+```
+
+You can list requests using `tsh request ls`.
+
+```code
+$ tsh request ls
+# Token                                Requestor Metadata       Created At (UTC)    Status
+# ------------------------------------ --------- -------------- ------------------- -------
+# bc8ca931-fec9-4b15-9a6f-20c13c5641a9 alice     roles=dba      07 Nov 19 19:38 UTC PENDING
+```
+
+## Reviewing requests
+
+In OSS Teleport, Access Requests must be reviewed by a cluster administrator
+with the ability to run `tctl` on the Auth Server.
+
+Administrators can list requests with `tctl requests ls`, and view the details
+of a particular request with `tctl requests get`.
```

Comment:
I think we should mention here that `tctl requests get` requires the request token.